### PR TITLE
#195

### PR DESCRIPTION
**Pull request by adireddy**

### CODE REVIEW

File: src/checkstyle/checks/whitespace/LineCheckBase.hx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+	public function new() {
+		super(LINE);
+		quotesRE = ~/('|")/;
+		multilineStartRE = null;
+		escapeRE = null;
+		multilineStringStart = false;
+	}
+
+	function isMultineString(line:String):Bool {
```
Comment (Gama11) [inline src/checkstyle/checks/whitespace/LineCheckBase.hx:19]: How can you reliably tell whether something is within a _multiline_ string based on a _single_ line?

File: test/checks/comments/TODOCommentCheckTest.hx [hunk 2]
```diff
@@ -56,4 +60,10 @@ abstract TODOCommentCheckTests(String) to String {
 		// XXX remove test
 		public override function test() {}
 	}";
+
+	var TEST6 = "
+	class Test {
+		var a:String = 'TODO: remove test';
```
Comment (Gama11) [inline test/checks/comments/TODOCommentCheckTest.hx:66]: I'm not sure this is a good test case, this is just a single line string.

**Comment by Gama11** [comment]
No test for `IndentationCharacterCheck`?

File: test/checks/comments/TODOCommentCheckTest.hx (hunk 2 above)
Comment (adireddy) [inline test/checks/comments/TODOCommentCheckTest.hx:66]: Not needed as TODO check is based on tokens so any string will be ignored.

File: src/checkstyle/checks/whitespace/LineCheckBase.hx (hunk 1 above)
Comment (adireddy) [inline src/checkstyle/checks/whitespace/LineCheckBase.hx:19]: Not sure how weird the strings can be but covered the basics if you see the test strings.

**Comment by adireddy** [comment]
Well removed `IndentationCharacterCheck` and `TabForAligningCheckTest` from exclude. Are there any more cases to test?

**Pull request merged**